package.json

```diff
@@ -78,7 +78,8 @@
     "hardhat-deploy": "^0.11.2",
     "hardhat-gas-reporter": "^1.0.8",
     "lodash.get": "^4.4.2",
-    "superstruct": "^0.15.4"
+    "superstruct": "^0.15.4",
+    "winston": "^3.8.1"
```

Comment:
FWIW this commit added the dependency on winston: https://github.com/across-protocol/sdk-v2/commit/ca0de27d29e84b4a6ea42bb4a325ada2c0c63121

Reply:
Just to wrap it up - I reverted this and added winston as a dev dependency instead.